Robot Code/src/org/usfirst/frc/team3316/robot/humanIO/SDB.java

```diff
@@ -72,21 +57,27 @@ public void run()
 			/*
 			 * Insert put methods here
 			 */
-			put("Intake Current", Robot.intake.getCurrent());
-			
 			put("Chassis Yaw", Robot.chassis.getYaw());
 			put("Chassis Pitch", Robot.chassis.getPitch());
 			put("isOnDefense", Robot.chassis.isOnDefense());
 
 			put("Hood Angle", Robot.hood.getAngle());
 			put("Turret Angle", Robot.turret.getAngle());
+			
+			put("Flywheel speed", Robot.flywheel.getRate());
+			put("Flywheel HE", Robot.sensors.flywheelHE.get());
+			put("Flywheel SC voltage", Robot.actuators.flywheelMotor.getVoltage());
+			put("Flywheel Current", Robot.actuators.flywheelMotor.getCurrent());
+			
+			put("Joystick Y", Robot.joysticks.joystickOperator.getY());
+			
 			try
 			{
 				//put("DistanceFromCamera", VisionServer.Data.get("DistanceFromCamera"));
```

Comment:
When we test the data from the camera we need to make sure this isn't commented out.